spec/facets/SchemaDatasetFacet.json

```diff
@@ -1,7 +1,38 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://openlineage.io/spec/facets/1-0-0/SchemaDatasetFacet.json",
+  "$id": "https://openlineage.io/spec/facets/1-1-0/SchemaDatasetFacet.json",
   "$defs": {
+    "SchemaDatasetFacetField": {
```

Comment:
Name of schema is used to generate Java (and hopefully) Python classes, and all of them are placed to one flat namespace (e.g. OpenLineage class in Java). This can introduce issues with future facets changes, class names may start to conflict.